Dockerfile.dist

```
@@ -80,12 +112,17 @@ LABEL \
   io.openshift.tags="rhtas rhtap trusted-artifact-signer trusted-application-pipeline enterprise-contract ec opa cosign sigstore" \
   com.redhat.component="ec-cli"
 
+# Install cosign and other tools we want to use in the Tekton task
 RUN microdnf -y --nodocs install git-core jq && microdnf clean all
```

One option as well is to set `--setopt=keepcache=0`

```suggestion
RUN microdnf -y --nodocs --setopt=keepcache=0 install git-core jq
```

---

Done.